File: packages/proximity-sensor/ios/Plugin/Classes/Results/IsAvailableResult.swift
import Capacitor

@objc public class IsAvailableResult: NSObject, Result {
    private let available: Bool

    init(available: Bool) {
        self.available = available
    }

    public func toJSObject() -> AnyObject {
        var result = JSObject()
        result["available"] = available
        return result as AnyObject
    }
}

File: packages/passkeys/android/src/main/java/io/capawesome/capacitorjs/plugins/passkeys/classes/results/IsAvailableResult.java
package io.capawesome.capacitorjs.plugins.passkeys.classes.results;

import androidx.annotation.NonNull;
import com.getcapacitor.JSObject;
import io.capawesome.capacitorjs.plugins.passkeys.interfaces.Result;

public class IsAvailableResult implements Result {

    private final boolean available;

    public IsAvailableResult(boolean available) {
        this.available = available;
    }

    @Override
    @NonNull
    public JSObject toJSObject() {
        JSObject result = new JSObject();
        result.put("available", available);
        return result;
    }
}
